istio.deps

```diff
@@ -4,6 +4,6 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "47d38c13e18d78f8079d602feba218138ce0fdab"
+    "lastStableSHA": "42c7f6d3c4328b5ae318421c7b332f48ccfe616b"
```

Comment:
this will fail without https://github.com/istio/istio/pull/25705

Reply:
You can remove proxy from this change. The 1.7 will use another branch for envoy anyway, so this is useless

Reply:
This change is reverted. Thanks